docker/android-slave-s2i/tools/android-wait-for-emulator.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
@wtrocki - Yes makes sense I'm not sure anyone would want to use the crappy emulator